scripts/src/blockchain/channelsGenerator.ts

```diff
@@ -0,0 +1,93 @@
+import {KeyPair} from "./keyPair";
+import * as crypto from "crypto";
+import {TxSender} from "./txSender";
+import {Network, Operation} from "@kinecosystem/kin-base";
+import {Environment} from "../environment";
+import {Server} from "@kinecosystem/kin-sdk";
+import {AccountDataRetriever, IAccountDataRetriever} from "./accountDataRetriever";
+import {BlockchainInfoRetriever, IBlockchainInfoRetriever} from "./blockchainInfoRetriever";
+
+export namespace Channels {
+
+	function createChannels(params: CreateChannelsParams): Promise<KeyPair[]> {
+		const server = new Server(params.environment.url);
+		Network.use(new Network(params.environment.passphrase));
```

Comment:
Why do you call to Network.use again?

Reply:
my thinking is, to have it as independent functionality (like Ron did in python), that doesn't require constructing `KinClient`, in this design, everything should be constructed and injected at this level.
The other option is, to expose a `ChannelGenerator` from `KinClient`, and then there will be a single constructing point at `KinClient`.
But as it's a seems like a util-sh kind of functionality, I exposed it this way, let me know what do you think